pandas/tests/io/test_gbq.py

```diff
@@ -19,7 +19,7 @@
 api_exceptions = pytest.importorskip("google.api_core.exceptions")
 bigquery = pytest.importorskip("google.cloud.bigquery")
 service_account = pytest.importorskip("google.oauth2.service_account")
-pandas_gbq = pytest.importorskip('pandas_gbq')
+pandas_gbq = pytest.importorskip("pandas_gbq", minversion="0.8.0")
```

Comment:
no
you need to support the stated min. version

Reply:
Why not bump the minimum version?

Reply:
Dropped. Now it'll error instead of skip if an old version of pandas-gbq is installed.

Reply:
Hmm where is this actually erroring?

Reply:
It'll import at the call to `pandas_gbq.read_gbq()` or `pandas_gbq.to_gbq()` with a "unexpected 'credentials' argument". There are a couple of versions of pandas-gbq that used `**kwargs` in `read_gbq()`. Unfortunately, it will just ignore the extra argument in those older versions.

Reply:
Do those kwargs begin at a certain version? Wondering if we shouldn't have test coverage for those to ensure behavior. Alternately could consider bumping the min version in our requirements files

Reply:
I couldn't find any explicit versions for `pandas-gbq` in any requirements files. https://github.com/pandas-dev/pandas/search?q=pandas-gbq&unscoped_q=pandas-gbq

kwargs for `read_gbq` were _removed_ in https://github.com/pydata/pandas-gbq/commit/e7007a883130116d6dc2814e6c859462f1fe10bf which corresponds to pandas-gbq version 0.5.0. Any version before that may silently fail.